src/schema/inputs/OrganisationInput.ts

```diff
@@ -24,8 +24,8 @@ export class BaseUpdateOrganisationInput {
   @Field(() => [UpdateNestedUserInput], { nullable: true })
   members?: UpdateNestedUserInput[];
 
-  @Field(() => [UpdateNestedTagInput], { nullable: true })
-  tags!: UpdateNestedTagInput[];
+  @Field(() => UpdateNestedTagsetInput, { nullable: true })
+  tagset!: UpdateNestedTagsetInput;
```

Comment:
tagset should be marked as nullable, not as required if the field is set to `nullable: true`